charts/kof-mothership/files/dashboards/cluster-deployment-events.yaml

```
@@ -558,7 +558,7 @@ templating:
       query: victoriametrics-logs-datasource
       queryValue: ''
       refresh: 1
-      regex: ''
+      regex: /^logs/
```

If the only datasource we allow here is named `logs` then it's better to anchor both sides of regex to avoid allowing also `logstash-regional-cluster` or similar:
```suggestion
      regex: /^logs$/
```